spec/javascripts/helper/reactionVariationsHelpers.js

```diff
@@ -21,4 +21,21 @@ async function setUpReaction() {
   return reaction;
 }
 
-export { setUpMaterial, setUpReaction };
+function getColumnGroupChild(columnDefinitions, groupID, fieldID) {
+  const columnGroup = columnDefinitions.find((group) => group.groupId === groupID);
+  const columnDefinition = columnGroup.children.find((child) => child.field === fieldID);
+
+  return columnDefinition;
+}
+
+function getColumnDefinitionsMaterialIDs(columnDefinitions, materialType) {
```

Comment:
small example to make clear that group data structure names are complex

Reply:
See 819bf93.